tests/helpers/package_available.py

```diff
@@ -22,6 +21,7 @@ def _package_available(package_name: str) -> bool:
 _FAIRSCALE_AVAILABLE = not _IS_WINDOWS and _package_available("fairscale")
 
 _WANDB_AVAILABLE = _package_available("wandb")
+_OPENPOINTS_AVAILABLE = _package_available("src.models.classification.openpoints_module")
```

Comment:
Once openpoints becomes a seperate module we'll want to update this

Reply:
Yehhh it's pretty ugly but works haha